EnvironmentSimulator/Libraries/esminiRMLib/esminiRMLib.cpp

```diff
@@ -239,6 +239,28 @@ extern "C"
 		return (int)(position.size() - 1);  // return index of newly created
 	}
 
+    RM_DLL_API int RM_IsRoadSuccessor(int road_id, int other_id)
+    {
+        if (odrManager == nullptr)
+        {
+            return -1;
+        }
+        else
+        {
+            roadmanager::Road *road = odrManager->GetRoadById(road_id);
+            roadmanager::Road *successor_road = odrManager->GetRoadById(other_id);
+
+            if(road->IsSuccessor(successor_road))
+            {
+                return 1;
+            }
+            else
+            {
+                return 0;
+            }
```

Comment:
Here I would have done like this instead but doesn't really matter, just cleaner in my opinion

```suggestion
            return (int)road->IsSuccessor(successor_road)
```